README.md

```diff
@@ -90,14 +85,9 @@ and health-checking mechanisms.
 
 ### Build and install `hare` rpm from source.
 **NOTE: If you have built Motr and HARE from source you will not need to generate RPM packages below**
```

Comment:
**NOTE: If you have built Motr and HARE from sources you need not generate RPM packages as below, however, it might be more convenient to build and install rpms on a multinode setup sometimes**

Reply:
Resolved these comments as part of #1610